client/jetpack-cloud/components/layout/header.tsx

```diff
@@ -1,6 +1,9 @@
+import classNames from 'classnames';
 import { Children, ReactNode } from 'react';
+import useDetectWindowBoundary from 'calypso/lib/detect-window-boundary';
 
 type Props = {
+	showStickyContent?: boolean;
```

Comment:
[non-blocker] It seems more natural to describe the modifier as 'isSticky' rather than 'showStickyContent'. I've seen other components use the first one.

Reply:
I was thinking the same. Then, I decided to keep it as showStickyContent since this value mainly conveys if the sticky content has to be displayed or not rather than saying if the content is sticky, which is decided in the LayoutHeader itself.